src/File.ts

```diff
@@ -24,6 +27,10 @@ export const initializeFile = ({
  * If you pass more than one replacement task, all subsequent tasks in the same
  * section must be re-rendered, as their section indexes change. Assuming that
  * this is done faster than user interaction in practice.
+ * 
```

Comment:
Whoops! my bad. I deleted the hooks because they were annoying .... then forgot to manually run them. I was only running the tests ... should be fixed now!